### ISSUE TITLE
[Bug]: Codelite 17 fails to create example hello world program on Linux Mint 21

### ISSUE THREAD

**Issue by oliverhbailey** [eranif/codelite#3177]
### What happened?

Went through all the steps to install all required tools listed and installed CodeLite 17.  Walked through example as provided, step-by-step and Error 5 is generated.  No object file is placed in the demo project debug folder.  In addition, creating a new release does nothing and nothing can be edited in any release in the way of any features or variables.

I then went to a windows machine and downloaded and installed Codelite 17 which worked perfectly.  I then went to Manjaro and installed CodeLite 17 from the current Arch/Manjaro repository and it wors fine.  This problem only exists on Mint 21.

Since it works on the other systems I installed on, I am not going beyond reporting the issue.  But there clearly is a problem with Mint 21 and  CodeLite 17 that someone in development needs to pursue further.

### Version

Other

### Operating system

Linux

### Steps to reproduce

```text
Exacts steps for Ubuntu/Mint/Debian on CodeLite web site for version 17.  No errors during the installation of CodeLite or required tools and libraries.  Every step was checked and nothing is missing.  CodeLite installed without errors.

Note: Codelite 13 was installed, removed, and purged because it experienced the same issue.  When it was purged, all required tools and libraries not used by another application were removed when told to do so by apt, before codelite 17 was installed.
```

**Comment by dghart** [comment]
I've just installed Codelite 17 on a Mint 21.1 VirtualBox guest. The installation process went normally and CodeLite works as expected.
I then set up a new workspace and added a C++ project. The project built and ran normally.

Perhaps I don't understand correctly what you mean by 'Hello world program'. Which program are you trying to run? Where did you get it from? In which dir is it?

Are you sure you are not just missing a program needed for compilation e.g. 'gcc' or 'make'?

**Comment by oliverhbailey** [comment]
No, there are no files missing.  In fact, I do work in a number of
languages and a number of the other language compilers got scanned in on
Windows 10 as GNU C++ compilers.  This includes two Free Pascal Cross
Compilers (FPC) which have nothing to do with CodeLite.  This was after an
auto update of CodeLite on Windows 10.  Incidentally, CodeLite dark theme
affected all the other running programs and forced them into dark menu mode
as well, and these were applications running before I started CodeLite.
That forced me to uninstall CodeLite which is just not performing as
expected and appears to be poorly tested.
I have removed CodeLite from all the machines at this time.  I liked prior
versions of CodeLite because it was fast and easy on resource, but it also
handled cross compiling well for a time.  Now it is riddled with problems
and I don't have time to go through it and do detailed reports.  Thank you
for your follow up, I still have Code:Blocks installed and have already
moved my projects back.  I hope CodeLite eventually gets back to well
behaving and being functional in cross platform development.

Thank you again.

Regards,,
Oliver Bailey
